source/StartConditionsPanel.cpp

```diff
@@ -0,0 +1,275 @@
+#include "Files.h"
+#include "FillShader.h"
+#include "Font.h"
+#include "FontSet.h"
+#include "Format.h"
+#include "GameData.h"
+#include "GameWindow.h"
+#include "Information.h"
+#include "Interface.h"
+#include "LoadPanel.h"
+#include "MainPanel.h"
+#include "Messages.h"
+#include "PlayerInfo.h"
+#include "Planet.h"
+#include "Preferences.h"
+#include "Rectangle.h"
+#include "Ship.h"
+#include "ShipyardPanel.h"
+#include "StarField.h"
+#include "System.h"
+#include "UI.h"
+#include "WrappedText.h"
+
+#include <SDL2/SDL.h>
+
+#include "gl_header.h"
+
+using namespace std;
+
+StartConditionsPanel::StartConditionsPanel(PlayerInfo &player, UI &gamePanels, LoadPanel *loadPanel)
+	: player(player), gamePanels(gamePanels), loadPanel(loadPanel)
+{
+	if(GameData::Start().size())
+	{
+		chosenStart = GameData::Start()[GameData::Start().size()-1];	
+		hasChosenStart = true;
+	}
+	
+	descriptionBox = GameData::Interfaces().Get("start conditions menu")->GetBox("start description");
+	entryBox = GameData::Interfaces().Get("start conditions menu")->GetBox("start entry");
+	entryListBox = GameData::Interfaces().Get("start conditions menu")->GetBox("start entry list");
+	entryInternalBox = GameData::Interfaces().Get("start conditions menu")->GetBox("start entry internal");
+
+}
+
+
+void StartConditionsPanel::Draw()
+{
+	glClear(GL_COLOR_BUFFER_BIT);
+
+	Information info;
+
+	string descriptionText; // String that will be shown in the description panel
+
+	if(hasChosenStart){
+		info.SetCondition("chosen start");
+		if(chosenStart.GetSprite())
+			info.SetSprite("start sprite", chosenStart.GetSprite());
+		info.SetString("name", chosenStart.GetName());
+		info.SetString("description", chosenStart.GetDescription());
+		info.SetString("planet", chosenStart.GetPlanet()->Name());
+		info.SetString("system", chosenStart.GetSystem()->Name());
+		info.SetString("date", chosenStart.GetDate().ToString());
+		info.SetString("credits", to_string(chosenStart.GetAccounts().Credits()));
+
+		descriptionText = chosenStart.GetDescription();
+	}
+	else if (!GameData::Start().size())
+	{
+		descriptionText = "No start scenarios were defined!\n\nMake sure that you installed Endless Sky and all of your plugins properly";
+	}
+	else
+	{
+		descriptionText = "";		
+	}
+
+	GameData::Background().Draw(Point(), Point());
+	GameData::Interfaces().Get("menu background")->Draw(info, this);
+	GameData::Interfaces().Get("start conditions menu")->Draw(info, this);
+	GameData::Interfaces().Get("menu start info")->Draw(info, this);
+	
+	const Font &font = FontSet::Get(14);
+
+	WrappedText *text = new WrappedText(font);
+
+	text->SetAlignment(WrappedText::LEFT);
+	text->SetWrapWidth(210);
+	text->Wrap(descriptionText);
+
+	// Only allow draws inside the description box
+	glScissor(
+		descriptionBox.Left() + GameWindow::TrueWidth() / 2,
+		descriptionBox.Top() + GameWindow::TrueHeight() / 2,
+		descriptionBox.Width(),
+		descriptionBox.Height());
+
+	glEnable(GL_SCISSOR_TEST);
```

Comment:
Remove use of glScissor _entirely_. I disagree that plugin authors should be writing novels to describe a custom start. As with planet descriptions and plugins' "about" text, it's the content author's _job_ to make sure their content looks good.

In a future PR, I'm open to refactoring how we get the game to draw text within specific user-definable-at-runtime bounds -- e.g. it would make a lot more sense for glScissor to be invoked directly by `WrappedText::Draw(const Rectangle &bounds, const Color &color)` or e.g. `Interface::TextBoxElement::Draw`